internal/collector/instance.go

```diff
@@ -146,13 +148,20 @@ func AddToPod(
 }
 
 // startCommand generates the command script used by the collector container
-func startCommand(includeLogrotate bool) []string {
+func startCommand(dir string, includeLogrotate bool) []string {
+	var mkdirScript string
+	if dir != "" {
+		mkdirScript = `
+` + shell.MakeDirectories(0o775, dir, dir+"/receiver")
```

Comment:
We had a few TODOs about creating the receiver directory through the pod, not through the collector, so I did this.

Reply:
Yeah, I was wondering if the logrotate struct might help with that... I was thinking to work some of that out in the pgadmin PR